ansible-tower-cli/README.md

```diff
@@ -0,0 +1,83 @@
+
+## Ansible Tower CLI Task
+
+[Ansible Tower](https://ansible.com/tower) (formerly ‘AWX’) is a web-based solution that makes Ansible even more easy to use for IT teams of all kinds, It provides the `tower-cli`([Tower-CLI]([https://docs.ansible.com/ansible-tower/latest/html/towerapi/tower_cli.html](https://docs.ansible.com/ansible-tower/latest/html/towerapi/tower_cli.html))) command line tool that simplifies the tasks of starting jobs, workflow jobs, manage users, projects etc.
+
+## Install the Task
+
+Install `tower-cli` task:
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/ansible-tower-cli/ansible-tower-cli-task.yaml
+```
+
+## Inputs `tower-cli-task`
+
+### Parameters
+
+* **ARGS:** args to execute which are appended to `tower-cli` e.g. `user-list` (_default_: `--help`)
+* **SSLVERIFY:** args to disable Tower SSL verification (_default_: `false`)
+* **HOST:** The Ansible Tower HOST
+* **tower-secret:** the name of the secret containing the user and password for authentication on the ansible tower host (_default_: `tower-creds`)
+
+## Tower-Secret
+
+This is a [secret]([https://kubernetes.io/docs/concepts/configuration/secret/](https://kubernetes.io/docs/concepts/configuration/secret/)) example with USER and PASS information that are going to be used by the task
+
+```
```

Comment:
you could use the following

```
```yaml
```

Reply:
I actually Didn't know about that, thank you for the tip :+1: